### ISSUE TITLE
How to get intermediate layer output from inception model using tenlsorflow serving

### ISSUE THREAD

**Issue by RahulGupta-Reflektion** [tensorflow/serving#358]
Hi,
I was looking at[ inception_clinet.py](https://github.com/tensorflow/serving/blob/master/tensorflow_serving/example/inception_client.py) to use Tensorflow serving as a part of the [tutorial](https://tensorflow.github.io/serving/serving_inception.html). Is it possible to use tensorflow serving architecture (using gRPC protocol)  to get features from intermediate layers(e.g. 'pool_3:0') ?

Somebody raised a similar issue here. But the answer doesn't talk about how to do it using tensorflow serving architecture. 
https://github.com/tensorflow/serving/issues/338

Correct me if I am wrong. I believe 
`prediction = sess.graph.get_tensor_by_name("InceptionV3/Logits/Predictions:0")`
would bypass the TensorFlow serving architecture.

**Comment by chrisolston** [comment]
You can create a signature which specifies the intermediate tensors you want to read as outputs, and then use the Predict API (see tensorflow_serving/apis/predict.proto).

**Comment by chrisolston** [comment]
(Sorry, didn't mean to close the issue -- pushed the wrong button.)

**Comment by gautamvasudevan** [comment]
Closing due to staleness. If this is still an issue, please file a new updated issue with current steps to reproduce. Thanks!
